micro_dl/inference/readme.md

```diff
@@ -83,4 +87,16 @@ sample_prediction = torch_predictor.predict_image(sample_input)
 
 ```
 
-#TODO: evaluation script and config
\ No newline at end of file
+## Exporting models to onnx
```

Comment:
The deployment script uses inference module and I am seeing the same error as above: 
```
(lit-monai) [shalin.mehta@gpu-a-2 microDL]$ python micro_dl/cli/onnx_export_script.py --model_path /hpc/projects/CompMicro/projects/virtual_staining/models/phase2nuclei018/lightning_logs/20230514-003340/checkpoints/epoch=29-step=4350.ckpt --stack_depth 5 --export_path /hpc/projects/CompMicro/projects/virtual_staining/models/phase2nuclei018/deployment/20230514-003340_epoch29.onnx 
Initializing model in pytorch...
Traceback (most recent call last):
  File "/hpc/mydata/shalin.mehta/code/microDL/micro_dl/cli/onnx_export_script.py", line 177, in <module>
    main(args)        
  File "/hpc/mydata/shalin.mehta/code/microDL/micro_dl/cli/onnx_export_script.py", line 164, in main
    export_model(model_dir, model_name, args.stack_depth, args.export_path)
  File "/hpc/mydata/shalin.mehta/code/microDL/micro_dl/cli/onnx_export_script.py", line 99, in export_model
    torch_predictor = inference.TorchPredictor(
  File "/hpc/mydata/shalin.mehta/code/microDL/micro_dl/inference/inference.py", line 59, in __init__
    self.network_z_depth = self.network_config["in_stack_depth"]
KeyError: 'in_stack_depth'
```